File: base-lib/src/main/java/cn/pompip/lib/entity/ArticleCategoryEntity.java
package cn.pompip.lib.entity;



import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.sql.Timestamp;

@Entity
public class ArticleCategoryEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String name;
    private int position;
    private Timestamp createTimestamp;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getPosition() {
        return position;
    }

    public void setPosition(int position) {
        this.position = position;
    }

    public Timestamp getCreateTimestamp() {
        return createTimestamp;
    }

    public void setCreateTimestamp(Timestamp createTimestamp) {
        this.createTimestamp = createTimestamp;
    }
}

File: src/main/kotlin/cn/pompip/myblog/entity/ArticleCategoryEntity.kt
package cn.pompip.myblog.entity

import java.sql.Timestamp
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
@Entity
data class ArticleCategoryEntity(
        @Id
        @GeneratedValue(strategy= GenerationType.IDENTITY)
        var id:Int,
        val name :String,
        val position:Int,
        val createTimestamp: Timestamp
)